docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -2,4 +2,173 @@
+
+
+sourced by ``make devshell``. When in doubt, you might
+
+.. code::
+
+   cat <<EOF >~/.condevrc
+   source ~/.bashrc
+   EOF
+
+
+Using a development shell
+=========================
+
+In an active development shell, the following ``make`` targets are
+available and act on all ``.py`` files under ``src/``:
+
+
++---------------------+------------------------------------------------------------+
+| Command             |  Description                                               |
++=====================+============================================================+
+| ``make format``     | Format with ``black`` and ``isort``                        |
++---------------------+------------------------------------------------------------+
+| ``make lint``       | Lint with ``pylint``                                       |
+|                     |                                                            |
++---------------------+------------------------------------------------------------+
+| ``make type check`` | Typecheck with ``mypy``                                    |
++---------------------+------------------------------------------------------------+
+| ``make unittest``   | Run unit tests and report coverage with ``pytest`` and     |
+|                     | ``coverage``                                               |
++---------------------+------------------------------------------------------------+
+| ``make test``       | Equivalent to                                              |
+|                     | ``make lint && make typecheck && make unittest``, plus     |
+|                     | checks defined CLI scripts                                 |
++---------------------+------------------------------------------------------------+
+
+
+Note that ``make format`` is never run automatically, to avoid
+reformatting under-development code in a way that might surprise the
+developer. A useful development idiom is to periodically run
+``make format && make test`` to perform a full code-quality sweep
+through the code.
+
+The order of the targets above is intentionally, and possibly useful:
+``make format`` will complain about certain kinds of syntax errors that
+would cause all the remaining code-quality tools to fail (and may change
+line numbers reported by other tools, if it ran after them);
+``make lint`` provides a good first check for obvious errors and
+anti-patterns in the code; ``make typecheck`` offers a more nuanced look
+at interfaces between functions, methods, etc. and may spot issues that
+would cause ``make unittest`` to fail.
+
+The ``make test`` command is also automatically executed when ``conda``
+builds a ``uwtools`` package, so it is important to periodically run
+these tests during development and, crucially, before merging changes,
+to ensure that the tests will pass when CI builds the ``workflow-tools``
+code.
+
+In addition to the ``make devshell`` command, two other ``make`` targets
```

Comment:
`two other` -> `three other`

Apparently this text deserves a good overall read. I'm doing so as part of this review, but you probably should, too, to make sure everything seems accurate and usefully stated to you, too. Others working on doc-related PRs should likely adopt a similar critical point of view when copying existing text from `README.md`.

Reply:
It would probably be better not to say "two" or "three" at all. Maybe "several"? Or just "other `make` targets"? This is an example of being unnecessarily specific.

Reply:
As an initial WIP to transfer over all documentation currently on our README, I agree it needs an overall read through. 

Now that the documentation has been transferred it's more obvious about where modifications can be added, what needs to be changed, and where the mistakes are.